spec/controllers/api/drug_stocks_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+RSpec.describe Api::DrugStocksController, type: :controller do
+  before do
```

Comment:
It seems like you're turning the flag on for all tests within the seed data, so is this just to ensure that the tests work when the user hasn't reseeded recently? Also, I don't see any uses of the feature flag in the code. Is the code behind this endpoint feature flagged?